pyproject.toml

```diff
@@ -1,10 +1,122 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "setuptools>=62.6",
     "setuptools_scm[toml]>=3.4.3",
     "wheel",
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "bilby"
+description = "A user-friendly Bayesian inference library"
+readme = "README.rst"
+requires-python = ">=3.10"
+license = { file = "LICENSE.md" }
+authors = [
+  { name = "Bilby Developers" }
+]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Science/Research",
+  "License :: OSI Approved :: MIT License",
+  "Programming Language :: Python :: 3",
+  "Topic :: Scientific/Engineering :: Physics"
+]
+dynamic = ["dependencies", "optional-dependencies", "version"]
+
+[project.urls]
+Homepage = "https://github.com/bilby-dev/bilby"
+Documentation = "https://bilby.readthedocs.io"
+Source = "https://github.com/bilby-dev/bilby"
+Tracker = "https://github.com/bilby-dev/bilby/issues"
+
+[project.scripts]
+bilby_plot = "cli_bilby.plot_multiple_posteriors:main"
+bilby_result = "cli_bilby.bilby_result:main"
+
+[project.entry-points."bilby.samplers"]
+"bilby.bilby_mcmc" = "bilby.bilby_mcmc.sampler:Bilby_MCMC"
```

Comment:
nitpic: I would use a formatter for TOML: indenting the sections would be performed automatically and would add clarity.

Reply:
Good idea, do you have any suggestions for a specific formatter?

Reply:
I've add this pre-commit hook: https://github.com/macisamuele/language-formatters-pre-commit-hooks?tab=readme-ov-file. It seems to work but doesn't indent this block.

Reply:
Interesting! I was using some formatter in VScode "Even better TOML formatter".